resources/certificate.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Author:: Blair Hamilton (<blairham@me.com>)
+# Cookbook Name:: windows
+# Provider:: cert
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+actions :add, :delete
+
+attribute :store_name, :kind_of => String, :name_attribute => true
+attribute :cert_id, :kind_of => String
+attribute :infile, :kind_of => String
```

Comment:
sure then if you don't specify a store_name it would use the path? I could see that as acceptable.